docset/winserver2022-ps/failoverclusters/Set-ClusterParameter.md

```diff
@@ -11,55 +11,69 @@ title: Set-ClusterParameter
 # Set-ClusterParameter
 
 ## SYNOPSIS
-Controls specific properties of an object in a failover cluster, such as a resource, a group, or a network.
+Controls specific properties of an object in a failover cluster, such as a resource, a group, or a
+network.
 
 ## SYNTAX
 
 ### NoMultiple (Default)
+
 ```
-Set-ClusterParameter [-InputObject <PSObject>] [-Create] [-Delete] [-Cluster <String>] [<CommonParameters>]
+Set-ClusterParameter [-InputObject <PSObject>] [-Create] [-Delete] [-Cluster <String>]
+ [<CommonParameters>]
 ```
 
 ### Single Parameter
+
 ```
-Set-ClusterParameter [-InputObject <PSObject>] [[-Name] <String>] [[-Value] <PSObject>] [-Create] [-Delete]
- [-Cluster <String>] [<CommonParameters>]
+Set-ClusterParameter [-InputObject <PSObject>] [[-Name] <String>] [[-Value] <PSObject>] [-Create]
+ [-Delete] [-Cluster <String>] [<CommonParameters>]
 ```
 
 ### Multiple Parameter
+
 ```
 Set-ClusterParameter [-InputObject <PSObject>] [[-Multiple] <Hashtable>] [-Create] [-Delete]
  [-Cluster <String>] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Set-ClusterParameter** cmdlet controls specific properties of an object in a failover cluster, such as a resource, a group, or a network. 
 
- -- For a disk resource, you can set the disk signature or GUID of a disk, and turn maintenance on or off for that disk. 
+The **Set-ClusterParameter** cmdlet controls specific properties of an object in a failover cluster,
+such as a resource, a group, or a network.
+
+ -- For a disk resource, you can set the disk signature or GUID of a disk, and turn maintenance on
+ or off for that disk.
```

Comment:
- For a disk resource, you can set the disk signature or GUID of a disk, and turn maintenance on
  or off for that disk.